### PR TITLE
Grandpa: save all completed prior rounds

### PR DESCRIPTION
**Pull request by marcio-diaz**
This PR saves all concluded rounds into DB.

TODO:
- I still want to think if this is the correct way to save all completed prior rounds or I need to take into account popped completed rounds when calling `complete()`.
- Maybe extend to answer queries.
- Add some CLI flags to query votes?

### CODE REVIEW

**Comment by arkpar** [comment]
Does this fix #3546?

**Comment by marcio-diaz** [comment]
> Does this fix #3546?

I think it can be used for that. But it was initiated by this comment of @rphmeier :

"About the database: at the moment we only persist some small amount of completed rounds. one thing we need to do soon is get a DB where we have _all_ prior completed rounds' data. This should also help with addressing any relevant issues

**Comment by gavofyork** [comment]
@rphmeier does this do what you suggested?

File: client/finality-grandpa/src/aux_schema.rs [hunk 1]
```diff
@@ -405,6 +406,18 @@ pub(crate) fn write_voter_set_state<Block: BlockT, B: AuxStore>(
 	)
 }
 
+/// Write completed round.
```
Comment (marcio-diaz) [inline client/finality-grandpa/src/aux_schema.rs]:
```suggestion
/// Write concluded round.
```

**Comment by gavofyork** [comment]
@marcio-diaz CI fails.

**Pull request merged**